### PR TITLE
tiled: Update to 1.10.1

### PR DESCRIPTION
**Pull request by podsvirov**

### CODE REVIEW

**Comment by Biswa96** [comment]
There is a new pull request with similar changes.

**Comment by podsvirov** [comment]
> There is a new pull request with similar changes.

Thanks. I see it. Now it's draft.

File: mingw-w64-tiled/PKGBUILD [hunk 1]
```diff
@@ -3,11 +3,11 @@
 _realname=tiled
 pkgbase=mingw-w64-${_realname}
 pkgname=("${MINGW_PACKAGE_PREFIX}-${_realname}")
-pkgver=1.10.0
+pkgver=1.10.1
 pkgrel=1
 pkgdesc='A general purpose tile map editor, built to be flexible and easy to use (mingw-w64)'
 arch=('any')
-mingw_arch=('mingw64' 'ucrt64')
+mingw_arch=('mingw32' 'mingw64' 'ucrt64')
```
Comment (MehdiChinoune) [inline mingw-w64-tiled/PKGBUILD:10]: qbs for mingw32 was removed.

Comment (podsvirov) [reply mingw-w64-tiled/PKGBUILD:10]: Almost. I caught the outgoing train :-)
But you are right. I'll just remove `mingw32`.
Did you notice any difficulties when porting to Qt6?

**Comment by podsvirov** [comment]
All checks passed. Ready to merge.

File: mingw-w64-tiled/PKGBUILD (hunk 1 above)
Comment (MehdiChinoune) [reply mingw-w64-tiled/PKGBUILD]: https://sourceforge.net/p/mingw-w64/bugs/968/
https://bugreports.qt.io/projects/QBS/issues/QBS-1724

**Pull request closed without merging**